examples/configurable/configurable.description

```diff
@@ -0,0 +1,12 @@
+// The configurable variables feature allows configuration of the module-level
+// variables at the program execution. This enables initializing
+// variables with values provided externally. The `configurable` keyword can
+// be applied to a module-level variable declaration that contains an initializer
+// for this purpose.<br/>
+//
+// The values of configurable variables can be provided in a file named
+// `configuration.toml` located in the running directory.
+// The configuration file must be in the TOML(v0.4) format.
+// The configured values would always be a subtype of `readonly` and `anydata`.
+// Currently, Ballerina supports this feature for the `int`, `float`,
+// `boolean`, and `string` basic types.
```

Comment:
We used the following for the release note

"Ballerina currently supports configurable variables of types `int`, `float`, `boolean`, and `string`. Future versions will add support for configurable variables of any type that is a subtype of `readonly&anydata`."

Shall we reuse the same?